src/datascience-ui/data-explorer/reactSlickGrid.tsx

```diff
@@ -265,10 +282,7 @@ export class ReactSlickGrid extends React.Component<ISlickGridProps, ISlickGridS
             // Save in our state
             this.setState({ grid, fontSize });
         }
-
-        // Act like a resize happened to refresh the layout.
```

Comment:
This change seems unnecessary? It's functionally equivalent.

Reply:
I had extracted some grid setup functionality into a `createSlickGrid` method to setup the grid on receiving a slice, but found this didn't work (caused problems with header row rerendering when setting columns). Submitting a new approach